### PR TITLE
Upgrade dep

### PR DESCRIPTION
**Pull request by mfleader**
## Changes introduced with this PR

- [X] upgrade dependencies to fix CVE
- [X] address linting

---
By contributing to this repository, I agree to the [contribution guidelines](https://github.com/arcalot/.github/blob/main/CONTRIBUTING.md).

### CODE REVIEW

File: connector.go [hunk 1]
```diff
@@ -107,7 +107,8 @@ func (c connector) Deploy(ctx context.Context, image string) (deployer.Plugin, e
 			_ = stdoutWriter.Close()
 			_ = stdinWriter.Close()
 		}()
-		_ = podExec.Stream(
+		_ = podExec.StreamWithContext(
+			ctx,
```
Comment (webbnh) [inline connector.go:111]: Was this change required in order to be able to update the dependencies?

**Review by webbnh — COMMENTED** [review]
The changes look fine, but I'm wondering if they are all related to upgrading the dependencies.  (I.e., if someone does a `git log`, are they going to be surprised at the contents?)

File: connector.go (hunk 1 above)
Comment (mfleader) [reply connector.go:111]: yes, the previous function, `Stream()` was deprecated in the newer version, and the linter demanded the deprecated stuff be removed

**Review by webbnh — APPROVED** [review]
👍
